dts/bindings/serial/atmel,sam-uart.yaml

```diff
@@ -15,3 +15,16 @@ properties:
       type: int
       description: peripheral ID
       required: true
+
+    pinctrl-0:
+      type: phandles
+      description: |
+        PIO pin configuration for URXD & UTRD signals.  We expect that
+        the phandles will reference pinctrl nodes.  These nodes will
+        have a nodelabel that matches the Atmel SoC HAL defines and
+        be of the form p<port><pin><periph>_<inst>_<signal>.
+
+        For example the USART0 on SAME7x would be
```

Comment:
I'm not sure how this is wrong, but you're referencing a USART in the atmel,sam-uart YAML, and a UART in the atmel,sam-usart YAML.   In the corresponding devicetree files both devices use the uart compatible.

Reply:
:), will fix that up.  copy/paste lots of iterations :)

Reply:
Fixed.